tests/functional/testplan/testing/multitest/test_multitest_parts.py

```diff
@@ -113,11 +113,14 @@ def test_multi_parts_invalid_parameter_1():
                   resource='MyPool')
 
     with log_propagation_disabled(TESTPLAN_LOGGER):
-        assert plan.run().run is False
+        result = plan.run()
```

Comment:
? Not sure that this change is necessary...

Reply:
Just for easy debugging